website/src/app/kb/authenticate/jumpcloud/readme.mdx

```diff
@@ -0,0 +1,229 @@
+connector that supports both authentication and directory sync. Use this guide
+if you're looking to setup SSO with JumpCloud for your Firezone account and
+optionally sync users and groups from JumpCloud to Firezone.
+
+<Alert color="info">
+  Directory sync is supported for the **Enterprise** plan only.
+</Alert>
+
+## Overview
+
+The JumpCloud connector uses OpenID Connect for authentication and JumpCloud's
+SCIM API to optionally support directory sync on Enterprise plans.
+
+When directory sync is enabled, users and groups are synced every few minutes to
+ensure that your Firezone account remains up-to-date with the latest identity
+data from JumpCloud. [Read more](/kb/authenticate/directory-sync) about how sync
+works.
+
+## Setup
+
+Setting up the JumpCloud connector is similar to the process of setting up a
+universal OIDC connector. The main difference is omission of the
+`Discovery Document URI` configuration field, as this is the same for all
+JumpCloud tenants.
+
+Follow the steps below to setup the JumpCloud connector.
+
+### Step 1: Start the JumpCloud provider setup in Firezone
+
+In your admin portal, go to `Settings -> Identity Providers` and click
+`Add Identity Provider`. Then, select `JumpCloud` from the list of identity
+providers.
+
+You'll be shown a summary of the steps you need to complete to setup the
+JumpCloud provider. Keep this page open as you'll need to refer to it in the
+following steps.
+
+### Step 2: Create a new SSO application in JumpCloud
+
+[Login to your JumpCloud admin portal](https://console.jumpcloud.com/login),
+click **SSO Applications** and then **+ Add New Application**.
+
+<Link
+  href="/images/kb/authenticate/jumpcloud/1-add-application.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/jumpcloud/1-add-application.png"
+    alt="SSO Applications"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Select **Custom Application** and click **Next**.
+
+<Link
+  href="/images/kb/authenticate/jumpcloud/2-custom-application.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/jumpcloud/2-custom-application.png"
+    alt="Custom Application"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+On the next screen, ensure **Manage Single Sign-On** is selected, and select the
+**Configure SSO with OIDC** option.
+
+<Link
+  href="/images/kb/authenticate/jumpcloud/3-select-options.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/jumpcloud/3-select-options.png"
+    alt="Select options"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Optionally, select **Export users to this app (Identity Management)** if you
+plan to set up directory sync for this connector.
+
+Click **Next**.
+
+<Link
+  href="/images/kb/authenticate/jumpcloud/3-select-options.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/jumpcloud/3-select-options.png"
+    alt="Select options"
+    width={1200}
+    height={1200}
+  />
+</Link>
```

Comment:
Looks like this image is repeated, was that intentional?

Reply:
Good catch, this is now fixed.